dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java

```diff
@@ -1,50 +1,97 @@
 package greencity.entity.table;
 
 import greencity.entity.user.employee.Employee;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
 
 class TableColumnWidthForEmployeeTest {
+    private static Employee employee;
+    private static TableColumnWidthForEmployee tableColumnWidth;
+    static int defaultWidth;
+
+    @BeforeAll
+    public static void setup() {
```

Comment:
I do not see any benefits of using `BeforeAll`
We can initialize all variables like this: 
```
private static Employee employee = Employee.builder().build();
    private static TableColumnWidthForEmployee tableColumnWidth  = new TableColumnWidthForEmployee(employee);
    private static int defaultWidth = 120;
```